server/src/main/scala/za/co/absa/atum/server/api/database/runs/functions/WriteCheckpoint.scala

```diff
@@ -57,8 +61,8 @@ class WriteCheckpoint(implicit schema: DBSchema, dbEngine: DoobieEngine[Task])
                   ${values.processStartTime},
                   ${values.processEndTime},
                   ${
-                    import za.co.absa.atum.server.api.database.DoobieImplicits.Jsonb.jsonbArrayPutUsingString
-                    measurementsNormalized.toList
+                    import za.co.absa.atum.server.api.database.DoobieImplicits.Jsonb.jsonbArrayPut
```

Comment:
Is this still needed? With import here?

Reply:
Yes. not to say that the import is sitting at the write position but yes, we still need it for being able to escape the any special characters.